### DISCUSSION TITLE
Ubuntu 24.04, is scheduled to be released on April 25, 2024.

### DISCUSSION THREAD

**Discussion by Patrikk-io** [regolith-linux/regolith-desktop#976] [General]
Are there plans to support Ubuntu 24.04 soon after release?

Thanks to all contributors for the great work. I've been really enjoying the Desktop over the last few months.

**Comment by kgilmer** [comment]
Hi @Patrick-Cairoli , thanks and yes we plan to support Ubuntu 24.04 and releases into the future.  I recently did a quick test and was able to run Regolith `unstable` on Noble and it seemed to work well.  Typically it takes us some time to test and patch up any regressions before release, so we typically trail an Ubuntu release by a month or two.
